### PR TITLE
Add an option to disable automatic dependent property changed. #282

### PR DESCRIPTION
**Pull request by LLeny**
#### Description
Added an option **TriggerDependentProperties** to the weavers.xml file, set to false will disable the dependency processor.
`<PropertyChanged TriggerDependentProperties='false'/>`

### CODE REVIEW

**Comment by ltrzesniewski** [comment]
Thanks! This looks good, the CI failure is unrelated, I'll fix this.

**Comment by ltrzesniewski** [comment]
I fixed the issue in master, but I can't trigger a rebuild in AppVeyor. 😕

**Comment by SimonCropp** [comment]
rebuild triggered

**Comment by ltrzesniewski** [comment]
Thanks! Hmm, I thought that AppVeyor would build a merge commit of the PR, but it's apparently not the case. I've rebased this PR.

**Comment by ltrzesniewski** [comment]
I think we can merge if you're OK with these changes @SimonCropp.

File: PropertyChanged.Fody/Config/EnableIsChangedPropertyConfig.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public bool EnableIsChangedProperty = true;
+
+    public void ResolveEnableIsChangedPropertyConfig()
+    {
+        var value = Config?.Attributes("EnableIsChangedProperty")
+            .Select(a => a.Value)
+            .SingleOrDefault();
+        if (value != null)
+        {
+            TriggerDependentProperties = XmlConvert.ToBoolean(value.ToLowerInvariant());
```
Comment (ltrzesniewski) [inline PropertyChanged.Fody/Config/EnableIsChangedPropertyConfig.cs:15]: That's a bad copy-paste 😉

**Comment by ltrzesniewski** [comment]
BTW @LLeny I think it would have been better to open a separate PR for that second config option, so we could discuss these changes independently. It's better to always open PRs from branches different than `master` because of stuff like this. Also, smaller PRs are easier to rewiew.

Let's leave this one here though, you just need to update the correct flag as I pointed out in the review.

**Comment by LLeny** [comment]
@ltrzesniewski Indeed I agree, that commit shouldn't have been part of this PR. I'll fix that copy paste, thank you for pointing it up.

**Comment by ltrzesniewski** [comment]
Thanks. Let's merge this, as there seem to be no objections.

**Pull request merged**